### DISCUSSION TITLE
Record Path Location

### DISCUSSION THREAD

**Discussion by tmg-mmartin** [almarklein/timetagger#486] [Q&A]
Does anyone know where the logs generated by record creation are stored? I've been trying to find these files. (I am hosting locally on a raspberry Pi)

**Comment by almarklein** [comment]
Logs are printed to stdout and not stored by default. If you're using docker, you can use `docker logs <container-name>`.

**Reply by tmg-mmartin** [reply]
> Logs are printed to stdout and not stored by default. If you're using docker, you can use `docker logs <container-name>`. 

Thanks for your reply! I guess I'm asking where in the file system can I find the "records?" I'm using python to run the program. But I assume there has to be a path the records are being stored to. What path are the records saved to when running locally using python?

**Reply by almarklein** [reply]
Ah, sorry, I thought you meant log-messages from the server. The sqlite databases are stored in the `datadir` (see https://timetagger.readthedocs.io/en/latest/libapi/). By default this is `~/_timetagger/users`.
